storage/elasticsearch_metadata.go

```
@@ -219,53 +273,83 @@ func (esc *BgMetadataElasticSearchConnector) sendAndClearBuffer() error {
 		}
```

if res.IsError is true there is the error silently dropped ? maybe log/update a metric ?

---

A counter is Incremented, I could add a log